### PR TITLE
Add rump publishing

### PR DESCRIPTION
**Pull request by ghost**
@RichardLitt this is live, give it a try <3

``` sh
> ssh pages@pages.ipfs.team ./create.sh $site
> ssh pages@pages.ipfs.team ./publish.sh $site $hash
```

Where `$site` is a name like `multiformats`, `ipld`, `blog`, or `ipfsio`, and `$hash` comes from `ipfs add`.

### CODE REVIEW

**Comment by ghost** [comment]
And the ipld.io and multiformats.io dnslinks are already wired to it

File: README.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+This repository falls under the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+
+### Want to hack on IPFS?
+
+[![](https://cdn.rawgit.com/jbenet/contribute-ipfs-gif/master/img/contribute.gif)](https://github.com/ipfs/community/blob/master/contributing.md)
+
+## License
+
+MIT
+>>>>>>> Add rump publishing
```
Comment (ghost) [inline README.md]: woops, stray merge conflict artifact

Comment (RichardLitt) [reply README.md]: Remove it? :)

Comment (ghost) [reply README.md]: removed it :)

**Comment by ghost** [comment]
> <lgierth> ok give it another try
> <lgierth> the issue was with $PATH not covering the golang directories, and i also added some logging and info output

added these fixes ^

**Comment by RichardLitt** [comment]
I just tried it:

```
ssh pages@pages.ipfs.team ./create.sh libp2p.io
created libp2p.io/
created libp2p.io/key
publishing initial hello-world page
pinning (a minority of api mismatch errors is okay)
pinning on /ip6/fc98:424c:b433:d7e2:7ee3:9541:73ff:2cdb/tcp/5001 ...
api version mismatch (/go-ipfs/0.4.3-rc4/ != /go-ipfs/0.4.3-dev/)
pinning on /ip6/fce3:c53b:c3c5:2f54:8bb0:b6d9:898e:f140/tcp/5001 ...
api version mismatch (/go-ipfs/0.4.4-dev/ != /go-ipfs/0.4.3-dev/)
pinning on /ip6/fcfe:eab4:e49c:940f:8b29:35a4:8ea8:b01a/tcp/5001 ...
api version mismatch (/go-ipfs/0.4.3-rc4/ != /go-ipfs/0.4.3-dev/)
pinning on /ip6/fc4e:5427:3cd0:cc4c:4770:25bb:a682:d06c/tcp/5001 ...
api version mismatch (/go-ipfs/0.4.3-rc4/ != /go-ipfs/0.4.3-dev/)
pinning on /ip6/fc3d:9a4e:3c96:2fd2:1afa:18fe:8dd2:b602/tcp/5001 ...
api version mismatch (/go-ipfs/0.4.3-rc4/ != /go-ipfs/0.4.3-dev/)
pinning on /ip6/fcd8:a4e5:3af7:557e:72e5:f9d1:a599:e329/tcp/5001 ...
api version mismatch (/go-ipfs/0.4.3-rc4/ != /go-ipfs/0.4.3-dev/)
pinning on /ip6/fc29:9fda:3b73:c1d2:9302:31e3:964c:144c/tcp/5001 ...
api version mismatch (/go-ipfs/0.4.3-rc4/ != /go-ipfs/0.4.3-dev/)
pinning on /ip6/fcdf:a296:afe3:7118:4135:cc0b:ff92:4585/tcp/5001 ...
api version mismatch (/go-ipfs/0.4.4-dev/ != /go-ipfs/0.4.3-dev/)
publishing ...
published https://ipfs.io/ipns/QmQZzSNcBQsG7StMqYSNJYj3vnyBcdK5saNu5CFKVu2yga
```

And got a page that said:

```
Hi there!

This is the default initial page of ipfs-pages.

For more information check https://github.com/ipfs/ipfs-pages
```

What do you think I did wrong?

**Pull request closed without merging**